internal/pkg/skuba/addons/dex.go

```diff
@@ -161,8 +161,12 @@ kind: Deployment
 metadata:
   name: oidc-dex
   namespace: kube-system
+  labels:
+    app: oidc-dex
+    caasp.suse.com/skuba-replica-ha: "true"
```

Comment:
I'm not familiar enough with skuba code, sorry if this is a dumb question.
If I have a cluster deployed with a previous version I will not have this new label, 
how is the upgrade handled in these cases? 
Will they keep working with the previous HA model or are we going to update customers to this new model?

Reply:
we have to update customers with the new model to include the label.

Reply:
how do we use to implement that, is this going to be part of a follow-up PR?

Reply:
Good question, I totally missed this part. Will look at how to apply through addon upgrades.

Reply:
If we add this label on the addon manifests that need it we can bump the manifest version and they should be automatically applied when running `skuba addon upgrade apply`.

Reply:
Bump up version and added logic to apply affinity with `skuba addon upgrade`.